docs/analytics/index.mdx

```diff
@@ -19,9 +19,9 @@ For more details on setting up Sourcegraph Analytics, see our [enablement instru
 
 ## Air-gapped Analytics
 
-Air-gapped customers will soon be able to use our self-hosted and locally deployed analytics service, built on Grafana, to see Sourcegraph usage data.
+<Callout type="info">Air-gapped analytics is an upcoming feature currently in development. If you would like to learn more, please get in touch with your Sourcegraph representative.</Callout>
```

Comment:
capitalize the product name "Air-Gapped Analytics" for consistency if that's how we are doing this